package/kernel/lantiq/ltq-vdsl-vr11-mei/patches/001-fix-compile.patch

```diff
@@ -0,0 +1,103 @@
+--- a/src/drv_mei_cpe_linux.h
++++ b/src/drv_mei_cpe_linux.h
+@@ -31,6 +31,9 @@
+ #include <linux/module.h>
+ 
+ #include <linux/sched.h>
++#if (LINUX_VERSION_CODE >= KERNEL_VERSION(4,11,0))
```

Comment:
I reckon the LINUX_VERSION_CODE #ifdef'ery can be removed here and below, we don't support older kernels.

Reply:
Done, handling git-less patches with incremental conflicts is way too painful though, please don't make me do that again :)